src/index.user.js

```diff
@@ -4547,25 +4547,30 @@ function hookUpDailyChallengeAutoCollect() {
     );
     claimAllButtonElement.style.float = "right";
     claimAllButtonElement.style.marginRight = "16px";
-    claimAllButtonElement.textContent = "Claim All";
+    claimAllButtonElement.textContent = "Claim All & Close";
     claimAllButtonElement.addEventListener("click", clickHandler);
     titleRootElement.appendChild(claimAllButtonElement);
 
     function clickHandler() {
       const claimButtonElements = document.querySelectorAll(
         ".missionWrapper:not(.alreadyClaimed) .claimButton, .rewardWrapper:not(.claimed) .claimButton"
       );
+      const closeBtn = document.querySelector('#dailyMissionsPopup > a.close.closeButton[title="Close"]');
       if (!claimButtonElements.length) {
         console.log("No more challenges to claim.");
+        closeBtn.click();
         return;
       }
       console.log(`Challenges to claim: ${claimButtonElements.length}`);
       for (let claimButtonElement of claimButtonElements) {
```

Comment:
Erep only allows clicking one of these buttons at once, so instead of looping over all elements we can click only the first one. I know it's not in your PR, but it would be very benefitial to include this change. Can you do this?